osu.Desktop/DiscordRichPresence.cs

```diff
@@ -94,6 +96,18 @@ private void updateStatus()
             {
                 presence.State = truncate(activity.Value.Status);
                 presence.Details = truncate(getDetails(activity.Value));
+
+                if (getOnlineID(activity.Value) != null)
+                {
+                    presence.Buttons = new Button[]
+                    {
+                        new Button() { Label = "Open Beatmap", Url = $"https://osu.ppy.sh/b/{getOnlineID(activity.Value)}" }
```

Comment:
I'm not sure we should be using this link style, as it's a legacy one. In theory the set ID should also be available in all cases the difficulty is, so I think we can form the full URLs here already?